docs/content/how-to/aws/create-aws-hosted-cluster-arm-workers.md

```diff
@@ -1,10 +1,66 @@
 ---
-title: Create ARM NodePools on AWS Hosted Cluster
+title: Create Arm NodePools on AWS Hosted Clusters
 ---
 
-# Create ARM NodePools on AWS Hosted Cluster
+# Create Arm NodePools on AWS Hosted Clusters
 
-## Creating a Hosted Cluster with ARM NodePool
+The `arch` flag was added to the NodePool Spec in OCP 4.14. The `arch` flag sets the preferred processor architecture for the NodePool (currently only supported on AWS).
+
+!!! note 
+
+    Currently, the only valid values for '--arch' are 'arm64' and 'amd64'. The HyperShift CLI will default to 'amd64' when the 'arch' flag is not specified by the user.
+
+## Creating Arm NodePools Through the API
+The hosted cluster custom resource (CR) must utilize a multi-arch manifested image. Here is an example with an OCP 4.14 multi-arch nightly image:
+```
+apiVersion: hypershift.openshift.io/v1beta1
+kind: HostedCluster
+metadata:
+  creationTimestamp: null
+  name: hypershift-arm
+  namespace: clusters
+spec:
+  autoscaling: {}
+...
+  release:
+    image: quay.io/openshift-release-dev/ocp-release-nightly@sha256:390a33cebc940912a201a35ca03927ae5b058fbdae9626f7f4679786cab4fb1c
+```
+
+The CPU architecture for a NodePool is specified through its CR spec: `spec.arch`. Here is an example:
+```
+apiVersion: hypershift.openshift.io/v1beta1
+kind: NodePool
+metadata:
+  creationTimestamp: null
+  name: hypershift-arm-us-east-1a
+  namespace: clusters
+spec:
+  arch: arm64
+  clusterName: hypershift-arm
+  management:
+    autoRepair: false
+    upgradeType: Replace
+  nodeDrainTimeout: 0s
+  platform:
+    aws:
+      instanceProfile: hypershift-arm-2m289-worker
+      instanceType: m6g.large
+      rootVolume:
+        size: 120
+        type: gp3
+      securityGroups:
+      - id: sg-064ea63968d258493
+      subnet:
+        id: subnet-02c74cf1cf1e7413f
+    type: AWS
+  release:
+    image: quay.io/openshift-release-dev/ocp-release-nightly@sha256:390a33cebc940912a201a35ca03927ae5b058fbdae9626f7f4679786cab4fb1c
+  replicas: 3
+status:
+  replicas: 0
+```
+
+## Creating a Hosted Cluster with Arm NodePool Through HyperShift CLI
```

Comment:
maybe worth mentioning that one can create more Arm nodepools other than default through the command `hypershift create nodepool`